lib/server/wilson.js

```diff
@@ -118,9 +118,19 @@ module.exports = function(app, appWilsonConfigJson, dependencies) {
 
   //WILSON CLIENT
   router.get('/:version?/client.wilson.js', function(req, res) {
+    //Set max age headers based on config for wilson (NOTE: maxAgeMs is millis, so we divide to specify seconds)
```

Comment:
Sounds good.  I'll open a ticket to add this, but will merge this in for the time being.